Makefile

```diff
@@ -31,13 +31,27 @@ Makefile.inc: configure
 	@echo "$@ generated. Run the make again."
 	@exit 1
 
-install: Makefile.inc
+hwdata.pc: hwdata.pc.in
+	datadir="$(datadir)"; \
+	if [ "$${datadir#$(prefix)}" != "$$datadir" ]; then \
+		datadir="\$${prefix}$${datadir#$(prefix)}"; \
+	fi; \
+	sed -e 's|@prefix@|$(prefix)|g' \
+		-e "s|@datadir@|$$datadir|g" \
+		-e "s|@pkgdatadir@|\$${datadir}/$(NAME)|g" \
```

Comment:
I'm a bit confused about the escaping here. The final result we want is something like:

```
prefix=/usr
datarootdir=${prefix}/share
pkgdatadir=${pc_sysrootdir}${datarootdir}/hwdata
```

Reply:
The final result:
```
prefix=/usr
datadir=${prefix}/share
pkgdatadir=${datadir}/hwdata

Name: hwdata
Description: Hardware identification and configuration data
Version: 0.360
URL: https://github.com/vcrhonek/hwdata
```

Good enough?

Reply:
Yeah sure, but what's up with all these `$$` and `\$$`? They're pretty confusing.

Reply:
`$$` will be replaced by single `$`.

Thus, 

(Makefile)`\$${var}` -> (shell)`\${var}` -> (result)`${var}`
(Makefile)`$$var` -> (shell)`$var` -> (result)value_of_var

Reply:
A few suggestions to improve this:

- Use single quotes instead of double quotes for pkg-config variable expansion: e.g. `-e 's|@pkgdatadir@|$${datadir}/$(NAME)|g'`
- Extract this snippet into a separate `.sh` file, so that there's no double-escaping and backslash-terminated lines going on anymore.

Reply:
> * Use single quotes instead of double quotes

Done

Reply:
Thanks!